bokehjs/src/lib/external/colab.d.ts

```diff
@@ -0,0 +1,24 @@
+declare interface CommMessage {
+  buffers: DataView[]
+  content: {
+    data: string
+  }
+}
+
+declare interface Comm {
+  messages: AsyncGenerator<CommMessage>
+}
+
+declare interface Kernel {
+  comms: {
+    register_target: (target: string, fn: (comm: Comm) => void) => void
+  }
+}
+
+declare interface googlecolab {
+  colab: {
+    kernel: Kernel | undefined
+  }
+}
+
+declare const googlecolab: googlecolab | undefined
```

Comment:
@mattpap I don't know how to deal with this, the actual interface is called `google` but if I override that it'll clash with the google maps interface defined elsewhere.

Reply:
Use
```ts
declare namespace google.colab {
  export const kernel: {
    comms: {
      register_target(target: string, fn: (comm: Comm) => void): void
    }
  }
}
```
This will utilize declaration merging to combine different definitions of `google`.

Reply:
Thank you!